### ISSUE TITLE
Some question about code implement!

### ISSUE THREAD

**Issue by chenxiangzZ** [hh23333/PVPM#21]
@hh23333 
hi ,when I read your code , I am confused about some places, I sincerely hope you can help me answer

1. i notice that your code didnot use rerank,Generally speaking, using rerank will improve the results，did you try it?

2.  in your paper ,the PVPM + Aug will improve ,did the 'Aug'  refers to the 'random flip' in the command?

3. when i use Openpose to extract pose imformation, i find it cannot get the skeleton while person suffers very severe occlusion ,will this pose_heatmap have influence to the model trainning?

4. i notice that you define v ∈ {0, 1}NP ,but in your referred paper CVPR 2015 "Subgraph Matching using Compactness Prior
for Robust Feature Correspondence " ,they define binary indicator vector:
![image](https://user-images.githubusercontent.com/30611183/96834609-da876400-1474-11eb-90f4-679c10f677b4.png)
is that mean: the i-th part  query should be corresponded with the i-th part  gallary, you dont considered the unaligned match?

5. i cannot understand why the  regularization term can "narrow down the scope of hyper-parameter selection ",can you explain it?

6. in the function IQP_solver, you use value_i = np.matmul(np.matmul(x,W),x)-(lambd*x).sum(), can this formulate the transpose of matrix X? i am not good at pytorch's grammer.
![image](https://user-images.githubusercontent.com/30611183/96836478-9cd80a80-1477-11eb-8717-17e2c4fce6cc.png)

7. in the algorithm of trianing PVPM:
![image](https://user-images.githubusercontent.com/30611183/96836436-90ec4880-1477-11eb-8d35-3164beb34659.png)
how to update parameters by three time using different losses in your code? i  just notice that there has one place to update loss

     `            loss.backward()
            self.optimizer.step()`

Thank you for your selfless rely before and taking the time to answer such simple questions
